server/controllers/finance/reports/generalLedger/index.js

```diff
@@ -35,20 +35,22 @@ function renderReport(req, res, next) {
   });
   let report;
   let data;
-  const currentDate = new Date();
 
   try {
     report = new ReportManager(REPORT_TEMPLATE, req.session, options);
   } catch (e) {
     return next(e);
   }
 
-  return Fiscal.getPeriodCurrent(currentDate)
+  let fiscal_year_id = options.fiscal_year_id;
```

Comment:
Let's ask @sfount - he made the original issue and has worked with the team at Vanga most recently.

@sfount what is your opinion?